content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-385.md

```diff
@@ -0,0 +1,58 @@
+---
+weight: 42
+title: 10.17.0.385
+layout: bundle
+---
+
+<!--10.17.0.372 - 10.17.0.385-->
+
+
+
+### Fixes
+
+<table>
+<colgroup>
+<col style="width: 15%;">
+<col style="width:50%;">
+<col style="width: 10%;">
+<col style="width: 12%;">
+<col style="width: 13%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>If a user drags the X axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

```

Comment:
```suggestion
<td>If a user drags the X-axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

```